src/error.rs

```diff
@@ -12,6 +12,8 @@ pub enum Error {
     DecompressingError(std::io::Error),
     /// An error occured when decoding a base64 encoded dataset.
```

Comment:
Maybe fix the previous misspellings of "occured" while you're at it. :-)

Reply:
Done :)